packages/sheets-formula/src/locale/function-list/cube/ru-RU.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default {
+    CUBEKPIMEMBER: {
+        description: 'Возвращает свойство ключевого показателя эффективности (KPI) и отображает имя KPI в ячейке. KPI — это количественная мера, такая как ежемесячная валовая прибыль или ежеквартальная текучесть сотрудников, используемая для мониторинга производительности организации.',
+        abstract: 'Возвращает свойство ключевого показателя эффективности (KPI) и отображает имя KPI в ячейке. KPI — это количественная мера, такая как ежемесячная валовая прибыль или ежеквартальная текучесть сотрудников, используемая для мониторинга производительности организации.',
+        links: [
+            {
+                title: 'Инструкция',
+                url: 'https://support.microsoft.com/en-us/office/cubekpimember-function-744608bf-2c62-42cd-b67a-a56109f4b03b',
```

Comment:
Maybe it would be better to use a link in Russian, such as https://support.microsoft.com/ru-ru/office/%D0%BA%D1%83%D0%B1%D1%8D%D0%BB%D0%B5% D0%BC%D0%B5%D0%BD%D1%82%D0%BA%D0%B8%D0%BF-744608bf-2c62-42cd-b67a-a56109f4b03b